### PR TITLE
patch for #2781: User Agent header in doc loader

### PR DESCRIPTION
**Pull request by gmulhearn-anonyome**
a patch for #2781 (see issue description). Longer term acapy may consider making a User Agent header globally for all requests, and perhaps configurable. however this one gets us by

### CODE REVIEW

**Comment by gmulhearn-anonyome** [comment]
hmmm, getting a pipeline failure; i dont know acapy CI enough to know if this normal...

**Comment by jamshale** [comment]
> hmmm, getting a pipeline failure; i dont know acapy CI enough to know if this normal...

Trying to re-run. Integration tests have been getting sporadic fails when there's multiple concurrent runs.

File: aries_cloudagent/vc/ld_proofs/document_downloader.py [hunk 1]
```diff
@@ -122,6 +123,7 @@ def download(self, url: str, options: Optional[Dict], **kwargs):
             headers = options.get("headers")
             if headers is None:
                 headers = {"Accept": "application/ld+json, application/json"}
+            headers["User-Agent"] = f"AriesCloudAgent/{__version__}"
```
Comment (gmulhearn-anonyome) [inline aries_cloudagent/vc/ld_proofs/document_downloader.py:126]: Pending more discussion in #2781

**Review by dbluhm — APPROVED** [review]
I'm thinking this is probably "good enough" to merge; future discussion can yield additional PRs but this will solve the immediate issue. I'd like to open another issue to capture the desire to apply `User-Agent` headers wherever we make outbound requests (probably just webhooks and message delivery?). I believe the integration test failure is unrelated (though a sanity check from @jamshale would be appreciated). I'm willing to merge with that one failing test.

**Pull request merged**